ksql-parser/src/main/java/io/confluent/ksql/parser/SqlFormatter.java

```diff
@@ -396,45 +396,34 @@ protected Void visitInsertValues(final InsertValues node, final Integer context)
       builder.append(" ");
 
       if (!node.getColumns().isEmpty()) {
-        maybeWrapInParens(node.getColumns().size() > 1, () -> {
-          final List<String> columns = node.getColumns();
-          for (int i = 0; i < columns.size(); i++) {
-            final String column = columns.get(i);
-            builder.append(column);
-            if (i < columns.size() - 1) {
-              builder.append(", ");
-            }
+        builder.append("(");
+        final List<String> columns = node.getColumns();
+        for (int i = 0; i < columns.size(); i++) {
+          final String column = columns.get(i);
+          builder.append(column);
+          if (i < columns.size() - 1) {
+            builder.append(", ");
           }
-        });
-        builder.append(" ");
+        }
+        builder.append(") ");
```

Comment:
Suggestion: You can convert this block in 3 lines with:
 ```
builder.append("(");
builder.append(String.join(",", node.getColumns()));
builder.append(")");
```

Reply:
nice! I'll do the same for the one below with a streams map